dataland-frontend/src/components/resources/frameworkDataSearch/ReferenceDataModelTranslations.ts

```diff
@@ -0,0 +1,48 @@
+export const companyDataNames = {
+  companyName: "Company Name",
```

Comment:
What does this file do?

Reply:
This file contains mappings:
- Mappings of the fields of the CompanyInformation model to the actual language words to be displayed in the company upload form.
- Mappings of the fields of the CompanyInformation model to the tooltip text that should be displayed for those fields.
- The sectors that should be available in the create company upload form.

BUT wouldn't it make sense to implement these options as enum in the backend too? What do you think?

Reply:
This is, as Emanuel wrote, mostly a mapping from backend variable names to display names. From my point of view that belongs in the frontend. The possible values for the GICS sectors is actually relevant for the backend as well and moving it to the backend makes sense. So it should be included in the openAPI definition in the end and we should only accept those values for uploads via API as well.

Reply:
I just checked if it's possible to implement this als enum in the backend and I noticed that in the Companies-Excel (latest version from 17th of January 2023) there is one sector called "Financial Services", which is called "Financials" in this mapping file here.
Either the Excel, or this mapping file needs to adjust to make it consistent. Otherwise the csv converter won't recognize this right?

And should the implementation of the GICS sectors in the backend happen in this ticket, or in another story? What do you think?

Reply:
The missmatched sector name should be aligned.
The GICS list was introduced in this story or not? Up until now we accepted every string for sector in the API upload process. So i would see it more as its own story, except it is not much effort to align the backend to GICS list.

Reply:
Yes the GICS list was introduced in this story.
I think it's quite an effort to make the backend use the GICS list as enum for sectors.
You'd have to touch the csv converter and also some tests, which currently only work because sector is a free text field.
That's why I vote for doing this in an own story.
I also don't want to endanger finishing this story until Tuesday.

Reply:
For me it would be fine to do it as its own story. 
What do the others think? (@SiegfriedSobkowiak , @fsoffel , @MarcinStanczyk7 )

However, in any case the sector name should be aligned.

Reply:
That would be fine for me, too.

Reply:
Sector name is fixed.

Reply:
for me, too.

Reply:
I created a backlog item for this at the bottom of the backlog so we can discuss the priority on Monday

Reply:
Ok, then i resolve